drop/download_data.sh

```diff
@@ -3,15 +3,19 @@ set -e
 
 # get data
 resource_url="https://www.cmm.in.tum.de/public/paper/drop_analysis/resource.tar.gz"
-wget -Nc $resource_url
-tar -zxvf resource.tar.gz
-rm -rf Data
-mv resource Data
+wget -nc $resource_url
+if [ ! -d Data ]; then
+  rm -rf resource
+	tar -zxvf resource.tar.gz
+	mv resource Data
+else
+    echo "Data directory already exists, is not updated"
+fi
 
 # prepare data
 cd Data
+cp config_relative_wb1.8.yaml ../config.yaml
 python fix_sample_anno.py
```

Comment:
the demo needs some cleanup. the user should not see a fix_sample_anno.py script. nor any other temporary files.

Reply:
Will address this in a new DROP version